src/gmt_remote.c

```diff
@@ -659,6 +729,8 @@ GMT_LOCAL int gmtremote_refresh (struct GMT_CTRL *GMT, unsigned int index) {
 			if (!access (old_indexpath, F_OK))
 				remove (old_indexpath);	/* Remove old index file if it exists */
 		}
+		else 
+			GMT->current.io.new_data_list = true;	/* Flag that we wish to delete datasets older than entries in this file */
```

Comment:
Should ``gmt_end_module()`` reset this to false so that externals (MEX/Julia) do not inherit this value in posterior module runs?

Reply:
IT is set to falske nanoseconds later